FORMATTING_GUIDE.md

```diff
@@ -0,0 +1,389 @@
+
+<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/dev-tools-send.png" alt="Send request">
+```
+
+Do not place an image next to text or insert artificial line breaks using `<br>`. Otherwise, the text might render as aligned to the bottom of the image, with the image on the right. 
+
+If the image is under a list item, place it on a new line with a tab. For more examples, see [Lists with code snippets or images](#lists-with-code-snippets-or-images).
+
+## Labels
+
+You can use the following labels:
+
+* label-blue 
+* label-green
+* label-purple 
+* label-red 
+* label-yellow 
+
+Use a purple label to specify in which version an API was introduced:
+
+```
+# Alias
+Introduced 1.0
+{: .label .label-purple }
+```
+
+If we introduce a breaking change to an operation, add an additional label with a link to the release note for that breaking change:
+
+```
+## Get roles
+Introduced 1.0
+{: .label .label-purple }
+[Last breaking change 2.0](https://example.com)
+{: .label .label-red }
+```
+
+## Callouts
+
+You can use four levels of callouts:
+
+*  `{: .note}` blue
+*  `{: .tip }` green
+*  `{: .important}` yellow
+*  `{: .warning}` red
+
+Place a callout directly under the paragraph to which you want to apply the callout style.
+
+**Example**: 
+
+```
+In case of a cluster or node failure, all PIT data is lost.
+{: .note}
+```
+
+For a callout with multiple paragraphs or lists, use `>`:
+
+```
+>   ****PREREQUISITE****
+>
+>   To use a custom vector map with GeoJSON, install these two required plugins:
+>   * OpenSearch Dashboards Maps [`dashboards-maps`](https://github.com/opensearch-project/dashboards-maps_) front-end plugin
+>   * OpenSearch [`geospatial`](https://github.com/opensearch-project/geospatial_) backend plugin
+{: .note}
+
+```
+
+## Buttons
+
+You can use either `copy` or `copy-curl` includes for code snippets. The `copy` include places a **Copy** button on the code snippet, while the `copy-curl` include places both **Copy** and **Copy as cURL** buttons. Use the `copy-curl` include for API requests. If an API request is already in the cURL format, use the `copy` include.
+
+**Example of a `copy` include:**
+
+````
+```bash
+curl -XGET "localhost:9200/_tasks?actions=*search&detailed
+```
+{% include copy.html %}
+````
+
+
+**Example of a `copy-curl` include:**
+
+````
+```json
+PUT /sample-index1/_clone/cloned-index1
+{
+  "aliases": {
+    "sample-alias1": {}
+  }
+}
+```
+{% include copy-curl.html %}
+````
+
+## Adding pages or sections
+
+When adding a page or a section, make the `nav_order` of the child pages multiples of 10. For example, if you have a parent page `Clients`, make child pages `Java`, `Python`, and `JavaScript` have a `nav_order` of 10, 20, and 30, respectively. Doing so makes inserting additional child pages easier because it does not require renumbering existing pages.
```

Comment:
```suggestion
When adding a page or a section, make the `nav_order` of the child pages multiples of 10. For example, if you have a parent page `Clients`, make child pages `Java`, `Python`, and `JavaScript` have a `nav_order` of 10, 20, and 30, respectively. Doing so makes inserting additional child pages easier because it does not require you to renumber existing pages.
```